lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -59,18 +77,26 @@ def get_dir_vector(zdir):
 
 
 class Text3D(mtext.Text):
-    '''
-    Text object with 3D position and (in the future) direction.
-    '''
+    """
+    Text object with 3D position and direction.
+
+    Parameters
+    ----------
+    x, y, z
+        The position of the text.
+    text : str
+        The text string to display.
+    zdir : {'x', 'y', 'z', None, 3-tuple}
+        The direction of the text. See `.get_dir_vector` for a description of
+        the values.
+
+    Additional Parameters
+    ---------------------
+    **kwargs
+         Keyword arguments are passed on to `~matplotlib.text.Text`.
+   """
 
     def __init__(self, x=0, y=0, z=0, text='', zdir='z', **kwargs):
-        '''
-        *x*, *y*, *z*  Position of text
-        *text*         Text string to display
-        *zdir*         Direction of text
-
-        Keyword arguments are passed onto :func:`~matplotlib.text.Text`.
-        '''
```

Comment:
Question: If a subclass puts a docstring up in the class level (as opposed to the `__init__` level), but the parent class has a docstring in its `__init__`, which one shows up in sphinx? I haven't checked if that is the case here, just wondering.

Reply:
I haven't checked either. Just following the [numpydoc guide](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring)

>  The constructor (`__init__`) should also be documented here [class docstring], the Parameters section of the docstring details the constructors parameters.